src/pages/[author]/[name]/Project.tsx

```diff
@@ -30,7 +30,7 @@ const Project: React.FC = () => {
     () => Object.values(project?.nodes ?? []),
     [project?.nodes]
   )
-  if (loading || !project) return <div>LOADING</div>
+  if (loading || !project) return <div>{loading}</div>
```

Comment:
this should consume the context, and call `setLoadingModalState`

Reply:
wait no, since this case is covered by the contextProvider, we can just return `<></>`